website/content/docs/enterprise/lease-count-quotas.mdx

```diff
@@ -86,6 +86,21 @@ cluster through various
 [metrics](/vault/docs/internals/telemetry/metrics/core-system#quota-metrics)
 and through enabling optional audit logging.
 
+## Lease count quota exceeded
+
+When the Lease Count Quota is exceeded, Vault will emit a `429 - Too Many
+Requests`, indicating that new leases are in violation of the quota.
+
+```
+* lease count quota exceeded
```

Comment:
I don't know and am not an expert on this but you might need more of the error for it to show up in search results.

Reply:
This is an interesting point. I just tried searching on the Vercel preview and didn't get a hit. I'm not sure if the Vercel previews use the same search index as the deployed doc site though.

Reply:
Yeah, definitely a good call-out. I'll look into this, thanks @mladlow  (also thank you for the idea to include this in the docs in the first place). 😄

Reply:
I was thinking more for Google indexing than our indexing, but sadly don't have a good grasp of how it works.

Reply:
@schavis indicated that the best SEO trick would be to have the exact error message in a top-level heading. I assume having internal links will help with that as well! 🤞